src/main/java/teammates/storage/api/CommentsDb.java

```diff
@@ -406,12 +402,7 @@ public void deleteCommentsByStudentEmail(String courseId, String email) {
         Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, email);
 
         // student right now cannot be giver, so no need to&should not check for giver
-        List<Comment> recipientComments = this.getCommentEntitiesForRecipients(courseId,
-                                                       CommentParticipantType.PERSON, email);
-
-        getPm().deletePersistentAll(recipientComments);
-
-        getPm().flush();
+        getCommentsForRecipientsQuery(courseId, CommentParticipantType.PERSON, email).deletePersistentAll();
```

Comment:
The line break will make the `delete` keyword more visible, which is the most important word of the statement. Otherwise this statement looks like a query (it starts with `get`).